frontend/src/ui/context/PlaidContext.js

```diff
@@ -0,0 +1,20 @@
+import { createContext } from 'react'
+
+const PlaidContext = createContext({})
```

Comment:
@phahulin after modifying a lot of files, fixing the unit tests accordingly, and finally storing this in `localStorage`, I'm not really sure we need to use React's contetxt API for this... **it seems a little bit overkill**:

> What's nice about the context API is that you wrap the component that you want to make it aware of (consume) the context, so the component's API stays transparent and you save yourselft of passing something using "props" down the component's tree...

In our case, since we are storing the Plaid token on `localStorage` to make the Plaid's bank account page more user friendly _and we **only** access it there on a refresh_, we could probably go with a simple set of changes: 

> add the module that stores the Plaid token and use it internally on PlaidButton

Reply:
by "module" you mean a good-old js file imported into `PlaidButton` source file?

Reply:
yes, that's right

Reply:
I think it makes sense to do it in a simpler way

Reply:
All right, I'll start another branch/PR. Will leave this one open so we can compare both. Then we'll merge one and close the other accordingly. Thanks!